lib/dpl/providers/pages.rb

```diff
@@ -12,7 +12,10 @@ class Pages < Provider
       gem 'octokit', '~> 4.14.0'
       gem 'public_suffix', '~> 3.0.3'
 
-      opt '--github_token TOKEN', 'GitHub oauth token with repo permission', required: true, secret: true
+      required :github_token, :deploy_key
```

Comment:
one of?

Reply:
yep https://github.com/svenfuchs/cl#required-options